### DISCUSSION TITLE
Allow users to manage cards/bank accounts

### DISCUSSION THREAD

**Discussion by cybex-dev** [flutter-stripe/flutter_stripe#443] [Q&A]
_Please note: I don't want to open a bug report or feature request specifically as I don't see anything related but I may also have missed it_

**Question (possible feature request)**
Does `flutter_stripe` allow managing of saved cards, creating new cards directly using flutter i.e. not using one of preset payment functions/workflows?

My use for this would be to allow users to view saved payment methods, revise or remove payment methods or set default/preferred payment methods without performing a payment immediately - is this possible or is this a feature to be added?

**Comment by cybex-dev** [comment]
Thanks @jonasbark - any suggestion for this?

**Comment by jrwenigma** [comment]
I'm also interested to know this?

**Reply by cybex-dev** [reply]
There is currently no client support for this, it must be managed through your own API/cloud functions and passed back to your client
